From 229cea669f4e294816e65fa1f0a5ab2b0d1a2d07 Mon Sep 17 00:00:00 2001
From: Tim Lunn <tim@feathertop.org>
Date: Tue, 31 Oct 2023 21:19:28 +1100
Subject: [PATCH] ARM: dts: rockchip: rv1126: Add ethernet alias

Add alias for ethernet0 to pick up mac address from u-boot

Signed-off-by: Tim Lunn <tim@feathertop.org>
---
 arch/arm/boot/dts/rv1126.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 6ef34c33c..5b4ea29b9 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -20,6 +20,7 @@ / {
 	interrupt-parent = <&gic>;
 
 	aliases {
+		ethernet0 = &gmac;
 		i2c0 = &i2c0;
 		i2c2 = &i2c2;
 		serial0 = &uart0;
-- 
2.40.1

